openassessment/xblock/submission_mixin.py

```diff
@@ -432,7 +432,7 @@ def get_download_urls_from_submission(self, submission):
         if 'file_keys' in submission['answer']:
             file_keys = submission['answer'].get('file_keys', [])
             descriptions = submission['answer'].get('files_descriptions', [])
-            file_names = submission['answer'].get('files_name', [])
+            file_names = submission['answer'].get('files_name', submission['answer'].get('files_names', []))
```

Comment:
Can you confirm from the DB if the `files_name` key is used in any submission? Given the usage of `files_names` in other code places, I am assuming this could have been an unintentional mistake.

Reply:
I have verified in Prod database, records are present in DB by both `files_names` and `files_name`